api/Element.json

```diff
@@ -8378,54 +8378,6 @@
           }
         }
       },
-      "tabStop": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/tabStop",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/tabStop will need to be removed too.

Reply:
I'm creating a PR to remove this.